server/res/graph.html

```diff
@@ -13,62 +13,127 @@
       <div class="wrapper">
         <div id="tester"></div>
         <script>
```

Comment:
I'd prefer if we put this code into a file called graph.js. That way other htmls can use it. You will need to add a static definition for this in app.py